src/cfnlint/customRules/Rule.py

```diff
@@ -17,13 +17,44 @@ class Rule(object):
     prop = ''
     operator = ''
     value = ''
+    lineNumber = 0
+    error_level = 'W'
+    ruleSet = []
+    error_message = ''
```

Comment:
These are all class variables, right? it's a bit confusing that there are also instance variables with the same name

Reply:
Great point. Instance variables have been given a more descriptive name which is not shared with class variables.